examples/_web-components-template.html

```diff
@@ -262,5 +262,57 @@ <h2 class="ds-text-heading--2xl">Review example</h2>
         });
       });
     </script>
+
+    <h2>Modal Dialog example</h2>
+    <ds-button id="modal-toggle">Open Modal</ds-button>
+    <ds-modal-dialog
+      is-open="false"
+      root-id="modal-dialog"
+      alert="false"
+      size="wide"
+      backdrop-click-exits="true"
+    >
+      <div>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed accumsan diam vitae metus
+          lacinia, eget tempor purus placerat.
+        </p>
+        <div slot="heading">
+          <h2>This is the modal heading.</h2>
+        </div>
+        <div slot="actions">
+          <form method="dialog">
+            <ds-button type="reset" value="rstBtn">Reset</ds-button>
+            <ds-button type="submit" value="sbmBtn">Confirm</ds-button>
+          </form>
+        </div>
+      </div>
+    </ds-modal-dialog>
+
+    <script>
+      document.addEventListener('DOMContentLoaded', (event) => {
+        const modal = document.querySelector('ds-modal-dialog') as HTMLDialogElement;
+        const toggleButton = document.getElementById('modal-toggle');
+
+        // You can also open and close via the underlying dialog element's close() and showModal()
+        // methods.
+        //   const dialog = document.querySelector('dialog') as HTMLDialogElement;
+        //   if (dialog.open) {
+        //     dialog.close();
+        //   } else {
+        //     dialog.showModal();
+        //   }
+        // };
```

Comment:
Is this still something you want to have in here?

Reply:
It's probably not necessary